lib/lobstersbot/pluggable_connection.rb

```diff
@@ -13,14 +13,29 @@ def did_start_up
       @timers = Timers::Group.new
       @memory = PStore.new(config_dir('memory.pstore'), true)
       
+      @triggers = {}
+      # Initialize plugins
+      evaluate(:initialize)
+
       @timers.every(60) { evaluate(:frequently) }
       Thread.new { loop { @timers.wait } }
     end
     
     def channel_message(sender, channel, message)
```

Comment:
If we are going to change the way we handle messages to channel, can we move the other channel command over too?

Reply:
Do you mean, by handling commands with the `@triggers` hash? But that would mean adding some sort of priority system, because we probably want commands to be evaluated last.

Reply:
Yes.

Reply:
Done with the latest changes.